fall23/deepface/detectfaces.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    class FaceRecognition:
</pre><p>Class is called FaceRecognition, but are you just doing Face Detection here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-atfal-mafkoda-missing-children/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>